src/lib/services/invoice-account-addresses-service.js

```diff
@@ -0,0 +1,33 @@
+const InvoiceAccountAddress = require('../models/invoice-account-address');
+const invoiceAccountsConnector = require('../connectors/crm-v2/invoice-accounts');
+const DateRange = require('../models/date-range');
+
+const getInvoiceAccountAddressModel = (startDate, addressModel, agentCompanyModel, contactModel) => {
+  const invoiceAccountAddress = new InvoiceAccountAddress();
+  return invoiceAccountAddress.fromHash({
+    dateRange: new DateRange(startDate, null),
+    address: addressModel,
+    agentCompany: agentCompanyModel,
+    contact: contactModel
+  });
+};
+
+const createInvoiceAccountAddress = async (invoiceAccount, startDate) => {
```

Comment:
I think this might be more flexible for the future if its signature was:
```
const createInvoiceAccountAddress = async (invoiceAccount, invoiceAccountAddress, startDate) => {
```